lib/plug/cowboy.ex

```diff
@@ -303,7 +303,7 @@ defmodule Plug.Cowboy do
     [ref || build_ref(plug, scheme), transport_options, protocol_options]
   end
 
-  @default_stream_handlers [Plug.Cowboy.Stream]
+  @default_stream_handlers [Plug.Cowboy.Telemetry, Plug.Cowboy.Stream]
```

Comment:
We can probably kill `Plug.Cowboy.Stream` as long as we listen to the early errors telemetry event and log.